gem/lib/metasploit-payloads.rb

```diff
@@ -12,6 +12,10 @@ module MetasploitPayloads
   METERPRETER_SUBFOLDER = 'meterpreter'
   USER_DATA_SUBFOLDER   = 'payloads'
 
+  ENCRYPTED_PAYLOAD_HEADER = 'encrypted_payload_chacha20_v1'
```

Comment:
Not a blocker; Maybe we could use a binary format here - not strongly opinionated here